drivers/ruby/lib/func.rb

```diff
@@ -151,7 +151,7 @@ def ==(rhs)
     def do(*args, &b)
       a = ((@body != RQL) ? [self] : []) + args.dup
       if a == [] && !b
-        raise RqlDriverError, "Expected 1 or more arguments but found 0."
+        raise ReqlDriverError, "Expected 1 or more arguments but found 0."
```

Comment:
This is the same problem as in the JS driver https://github.com/rethinkdb/rethinkdb/issues/4646
Changing this is a separate issue.